src/index.css

```diff
@@ -0,0 +1,68 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #fff;
+  color: #000;
+  font-size: 0.89rem;
+}
+
+section {
+  padding: 2% 7%;
+}
+
+.section-title {
+  text-align: center;
+  font-size: 2rem;
+}
+
+.scores {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 1rem;
+}
+
+.score-list-section {
+  width: 50%;
+}
+
+.add-section {
+  width: 50%;
+}
+
+.refresh-title {
+  display: flex;
+  justify-content: space-around;
```

Comment:
- [ ] [OPTIONAL] Though styling is not required for this project, kindly update your stylesheet to make your project more user friendly and proffessional. Remember, your projects are a testimony of how good you are as a developer. :stuck_out_tongue_winking_eye: